src/conq/data_recorder.py

```diff
@@ -152,6 +158,20 @@ def save_episode_worker(self, clients, done: Event, mode: str, episode_save_inte
         # ensure we save before exiting or moving on to the next episode
         with episode_path.open('wb') as f:
             pickle.dump(episode, f)
+    
+    def copy_map_files(self, map_path: Path):
+        if map_path.exists():
+            name = str(map_path).split('/')[-1]
+            print(f"MAP PATH: {map_path}")
+            if map_path.is_dir():
+                shutil.copytree(map_path, f"{self.root}/{name}")
+                self.metadata['map_path'] = f"{self.root}/{name}"
+            else:
+                shutil.copyfile(map_path, f"{self.root}/{name}")
+                self.metadata['map_path'] = f"{self.root}/{name}"
+        else:
+            print(f"WARNING from data_recorder.py: map path {map_path} does not exist! Not adding path to metadata.")
+
```

Comment:
Since you're taking in a `Path` object and using Python >= 3.8, consider doing the following:

```python
		if map_path.exists():
            print(f"MAP PATH: {map_path}")
            dst = self.root / map_path.name
            if map_path.is_dir():
                shutil.copytree(map_path, dst)
            else:
                shutil.copyfile(map_path, dst)
			self.metadata['map_path'] = dst.as_posix()
        else:
            print(f"WARNING from data_recorder.py: map path {map_path} does not exist! Not adding path to metadata.")
```

Reply:
Good suggestion! Done in https://github.com/UM-ARM-Lab/conq_python/pull/35/commits/6b4d3c476b68ed75402a03d105fce243aa1a49d1